projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -19,12 +19,24 @@
     </a>
   </div>
   <div class="col-12 col-md-8">
-    <a
-      [routerLink]="{ cxRoute: 'product', params: product } | cxUrl"
-      class="cx-product-name"
-    >
-      <h2 [innerHtml]="product.nameHtml"></h2>
-    </a>
+    <ng-container *cxFeatureLevel="'6.8'">
+      <h2 class="cx-product-name">
+        <a
+          [routerLink]="{ cxRoute: 'product', params: product } | cxUrl"
+          [innerHtml]="product.nameHtml"
```

Comment:
Do you think it will be possible here to somehow render the html inside the h2 tag without using the `innerHtml` directive? 

The reason for this is to take advantage of the breaking change to make this component work with Angular 17's new feature "non-destructive" hydration. In which, we need to avoid manipulating the DOM to take advantage of this feature, and this is one of the places it doesn't work.

https://angular.io/guide/hydration#direct-dom-manipulation

Reply:
I've tested it quite a bit and it probably is not possible to keep the hydration benefits and render that HTML. Any direct DOM changes will cause a mismatch and I am not sure there is a way around it.

Reply:
No problem. It can be quite a challenge so thanks for taking a look at it. Supporting hydration is not currently a requirement (who know's if it will become one).